### ISSUE TITLE
what "capstone_static_winkernel" project is for?

### ISSUE THREAD

**Issue by Levimocun** [aquynh/capstone#956]
I compile the capstone in windows10. Now I've already compile the capstone_dll and capstone_static project successfully. And the test program also runs correctly. So i don't know why I also need to compile the capstone_static_winkernel ?

**Comment by aquynh** [comment]
that is to disassemble from inside Windows kernel. ignore that if you do not

**Comment by Levimocun** [comment]
@aquynh Could please tell the difference between disassembling from inside and outside the Windows kernel.

**Comment by aquynh** [comment]
You can see the code yourself.

One examole: memory allocation is different

**Comment by Levimocun** [comment]
@aquynh Now i couldn't compile the capstone_winkernel project successfully, so i can't test it by myself. Please answer me one more question, if i just disassemble the regular PE file , is there any difference between them?

**Comment by Levimocun** [comment]
I mean the difference at disassemble result

**Comment by aquynh** [comment]
Capstone is just a basic disassembler, so it cannot handle PE files. see projects using Capstone at http://www.capstone-engine.org/showcase.html to understand how you can write your own tools.

**Comment by Levimocun** [comment]
Yes, i know.  I've  extracted the binary code segment from PE file. I just want to know  there is any difference of disassemble result between disassembling inside and outside kernel. Thank you.

**Comment by aquynh** [comment]
No.

**Comment by Levimocun** [comment]
Ok, thank you very much.

**Comment by Kalfan** [comment]
Win32 code compiled with VS 2013 DOES NOT run on WinXP, this is a known issue.
As many people regard WinXP as a superior OS than Win7, can we then have the capstone_static_winkernel be set to be compilable by VS 2010 ?
